posixlib/src/main/scala/scala/scalanative/posix/sys/wait.scala

```diff
@@ -0,0 +1,10 @@
+package scala.scalanative.posix.sys
+
+import scala.scalanative.unsafe._
+
+@extern
+object wait {
+  @name("wait")
```

Comment:
I am really not understanding this exactly. I am seeing the following:
```
def wait(status: Ptr[CInt]: pid_t = extern
```
Reference: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/sys_wait.h.html

Note: `pid_t` is in sys/types.

Reply:
I think this is an attempt to solve the fact that the object & the method within it
have the same name.  Privately, I call this the "errno.errno" problem because I have
tripped over it so many times. My karma probably did not improve from all the bad
words I used on errno.errno.

There are a number of bad solutions.     I think the least bad is to do away with
the `@name`, use the expected `wait` as the method name and, importantly,
leave a one line comment above the method something like
`Use as wait.wait`.  I damn myself with my own writing....

Reply:
@LeeTibbert @ekrich it's actually worse than that - `wait()` collides with a method on java.lang.Object:
https://docs.oracle.com/javase/7/docs/api/java/lang/Object.html
So it can't be overridden, but I wanted to make the C wait() available anyway.

Reply:
@ekrich you are absolutely right that I totally bungled the signature of wait() lol.

Reply:
@rwhaling   I concur that making the C wait available is a worthwhile goal.

Aren' the method signatures different?  the C wait takes a pointer to a pid_t as
an argument and returns a pid_t.  I followed your URL (thank you) and the 
Java method(s) return void. 

As always, if I am out of order or out to lunch here, let me know.